force-app/main/educationCloudSettings/lwc/edcSettingsProductDisplay/edcSettingsProductDisplay.js

```diff
@@ -0,0 +1,48 @@
+import { LightningElement, api, track } from 'lwc';
+import stgHealthCheckLoadingIndicator from "@salesforce/label/c.stgHealthCheckLoadingIndicator";
+import { ShowToastEvent } from "lightning/platformShowToastEvent";
+
+export default class EdcSettingsProductDisplay extends LightningElement {
+    @api productRegistryVModels;
+
+    @track loadedSettingsProductCount = 0;
+    @track displaySettingsProduct = false;
+    errorList = [];
+
+    LabelReference = {
+        spinnerLoadingAltText: stgHealthCheckLoadingIndicator,
+    };
+
+    handleSettingsProductLoaded(evt) {
+        this.loadedSettingsProductCount++;
+        this.areAllProductsLoaded();
+    }
+
+    handleSettingsProductError(evt){
+        const errorMessage = evt.detail.errorMessage;
+        this.errorList.push(errorMessage);
+        this.loadedSettingsProductCount++;
+        this.areAllProductsLoaded();
+    }
+
+    areAllProductsLoaded(){
```

Comment:
This method provides a fantastic user experience and not something called out in the spec. Having the error messages come in once the tiles are loaded all at once provides a context for that error. This will play out with visual environments and with screen readers.

Reply:
Thanks for noticing that, I'll add a note in the spec. :-)